tests/integration/test_charm.py

```diff
@@ -84,18 +87,40 @@ async def test_build_and_deploy(ops_test: OpsTest, charm: str, test_charm: str)
 async def test_requirer_charm_integration(ops_test: OpsTest) -> None:
     assert ops_test.model.applications[OPENFGA_APP].status == "active"
 
-    await ops_test.model.integrate(OPENFGA_APP, "openfga-requires")
+    await ops_test.model.integrate(OPENFGA_APP, OPENFGA_REQUIRES_APP)
     async with ops_test.fast_forward():
         await ops_test.model.wait_for_idle(
-            apps=["openfga-requires"],
+            apps=[OPENFGA_REQUIRES_APP],
             status="active",
             timeout=60,
         )
 
-    openfga_requires_unit = ops_test.model.applications["openfga-requires"].units[0]
+    openfga_requires_unit = ops_test.model.applications[OPENFGA_REQUIRES_APP].units[0]
     assert "running with store" in openfga_requires_unit.workload_status_message
 
 
+async def test_loki_integration(ops_test: OpsTest) -> None:
+    logger.debug("deploying loki")
+    await ops_test.model.deploy(
+        LOKI_CHARM,
+        application_name=LOKI_APP,
+        trust=True,
+    )
+    await ops_test.model.wait_for_idle(
+        apps=[LOKI_APP],
+        status="active",
+        raise_on_blocked=True,
+        timeout=1000,
+    )
+    await ops_test.model.integrate(OPENFGA_APP, LOKI_APP)
+    async with ops_test.fast_forward():
+        await ops_test.model.wait_for_idle(
+            apps=[LOKI_APP],
+            status="active",
+            timeout=60,
+        )
```

Comment:
Not sure if this test makes much sense. 
We don't test the cos integration right now (but it would be nice to validate that it works in integration tests). BUT this test only validates that the openfga app can integrate with loki, not that the logs actually can reach loki. We can test the same thing through unit tests, with minimum overhead.

I would suggest to remove this test for now and we can investigate testing the cos integration in another issue.